packages/mui-base/src/Select/Select.test.tsx

```diff
@@ -1254,7 +1254,6 @@ describe('<Select />', () => {
     const selectButton = getByTestId('select');
 
     act(() => updateButton.click());
-    act(() => selectButton.click());
```

Comment:
This was throwing a "not wrapped in `act(...)`" warning. Not sure why. This click is not necessary for the test. As this is from the stalled base package, and it won't be published from this branch, I don't think it's worth digging deeper into why this was here and/or started throwing.